ansible/roles/ocp4-workload-idm/files/oauth-ldap.yaml

```diff
@@ -1,10 +1,11 @@
+---
 apiVersion: config.openshift.io/v1
 kind: OAuth
 metadata:
   name: cluster
 spec:
   identityProviders:
-  - name: {{ idm_domain_name }}
+  - name: {{idm_domain_name}}
```

Comment:
Why did we lose the spaces inside the brackets here?

Reply:
Yes I agree. As the opportunity arises it would be good to always insert spaces around the var names (and expand constructs like `d(foo)` to `default(foo)`, also spaces around `foo|bool` to `foo | bool`. All makes it more readable.

Reply:
I'm not sure .. yamllint was complaining about it. does it need to be in quotes (")?

Reply:
`./files/oauth-ldap.yaml
  8:13      error    too many spaces inside braces  (braces)
  8:29      error    too many spaces inside braces  (braces)`

That's what is says when I add the spaces back. Not sure why.

Reply:
Ah! I see the issue here. This is a jinja template, not a normal YAML file. The file name should end in .j2